adafruit_ov7670.py

```diff
@@ -631,7 +639,7 @@ def deinit(self):
             self._reset.deinit()
 
     @property
-    def size(self):
+    def size(self) -> Optional[int]:
```

Comment:
I think this one can be `int` without the `Optional` as well.

It's expecting one of the `OV7670_SIZE_` constants which are all ints.